src/ajitSJune20/BankingSystem.java

```diff
@@ -0,0 +1,64 @@
+package ajitSJune20;
+
+public class BankingSystem {
+
+	int debitCount = 0, creditCount = 0, printBalanceCount = 0, currentBalance = 0;
+	static int totalDebitCount = 0, totalCreditCount = 0, TotalPrintBalanceCount = 0;
```

Comment:
Remove the initialization of all instance variables.

Reply:
Incorporated changes